doc/semicons.xml

```diff
@@ -299,3 +299,32 @@ true]]></Example>
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="SemilatticeFromDigraph">
+  <ManSection>
+    <Func Name="SemilatticeFromDigraph" Arg="digraph"/>
+    <Returns>
+      A semilattice of partial perms <A>n</A>.
+    </Returns>
+    <Description>
+      If <A>digraph</A> is a
+      <Ref Prop="IsJoinSemilatticeDigraph" BookName="Digraphs"/> or a
+      <Ref Prop="IsMeetSemilatticeDigraph" BookName="Digraphs"/>, then this
+      function returns a semigroup of partial perms which is isomorphic to
+      the semigroup whose elements are the vertices of <A>digraph</A> with
+      the binary operation
+      <Ref Prop="PartialOrderDigraphJoinOfVertices" BookName="Digraphs"/>
+      or <Ref Prop="PartialOrderDigraphJoinOfVertices" BookName="Digraphs"/>,
+      respectively.
+      <Example><![CDATA[
```

Comment:
I definitely think you should have separate functions `SemilatticeFromJoinSemilatticeDigraph` and `SemilatticeFromMeetSemilatticeDigraph`, to avoid the confusion to give the user more control.